src/navigation/ActivationStack.tsx

```diff
@@ -24,35 +23,42 @@ type ActivationStackParams = {
 
 const Stack = createStackNavigator<ActivationStackParams>()
 
+interface ActivationStep {
+  screenName: ActivationScreen
+  component: FunctionComponent
+}
+
+const activateProximityTracing: ActivationStep = {
+  screenName: ActivationScreens.ActivateProximityTracing,
+  component: ActivateProximityTracing,
+}
+
+const notificationPermissions: ActivationStep = {
+  screenName: ActivationScreens.NotificationPermissions,
+  component: NotificationPermissions,
+}
+
+const activationStepsIOS: ActivationStep[] = [
+  activateProximityTracing,
+  notificationPermissions,
+]
+
+const activationStepsAndroid: ActivationStep[] = [activateProximityTracing]
+
+const activationSteps =
```

Comment:
I'm a bit concerned about having logic among the declarations of this file, is there another place we could do this?

Reply:
i agree its a bit weird, we can look for a better spot.

Reply:
To keep this pr small, gonna roll back the `firstActivationScreen` concept, which will allow this to live in the component. We lose the single source of truth for the route progression, but we can revisit handling the routing in a more thoughtful way in a future commit.